### ISSUE TITLE
Scrollbar:false is not working

### ISSUE THREAD

**Issue by NaveenkumarMD** [prabhuignoto/react-chrono#214]
```
        <Chrono
          items={items}
          mode="VERTICAL_ALTERNATING"
          hideControls
          scrollable={{ scrollbar: false }}
        />
```
I have the  hide the scrollbar .I followed the documentation but it's not working.Could you please have a look into this. 

![image](https://user-images.githubusercontent.com/72449129/149619327-66fce5ff-6fe9-4878-bf59-d7b33b8860fb.png)

**Comment by prabhuignoto** [comment]
@NaveenkumarMD thanks for reporting the issue. is there a public repo for me to look at

**Comment by NaveenkumarMD** [comment]
https://github.com/NaveenkumarMD/cryptera-website
The time line page in this website .

**Comment by prabhuignoto** [comment]
This is now fixed with  [1.13.1](https://github.com/prabhuignoto/react-chrono/releases/tag/1.13.1).

**Comment by NaveenkumarMD** [comment]
Thanks a lot ❤❤
